package.json

```diff
@@ -5,11 +5,12 @@
   "description": "Red Badger's website",
   "main": "build/server.js",
   "scripts": {
+    "clean": "rimraf build",
     "test:server": "BABEL_DISABLE_CACHE=1 BABEL_ENV=test mocha --compilers js:babel-register ./src/**/*.spec.js",
     "test:client": "karma start config/karma.config.js",
     "test:e2e": "wdio config/wdio.config.js",
     "start": "node build/server.js",
-    "build": "webpack --config config/webpack.config.js",
+    "build": "npm run clean && webpack --config config/webpack.config.js",
```

Comment:
Only asking because if it does, I think it would be better to just run this as a pre-dependency step and then we don't have to rely on an additional `node_module`